framework/include/media/MediaUtils.h

```diff
@@ -44,6 +110,40 @@ typedef enum audio_type_e {
 	AUDIO_TYPE_FLAC = 5
 } audio_type_t;
 
+/**
```

Comment:
Actually we don't need most of things, I think we need only 2 things we can expand later.. please see below
```
in mediautils.h
typedef enum pcm_format_e {
     AUDIO_PCM_BIT_S16_LE = 0,
     AUDIO_PCM_BIT_S32_LE = 1
}

```
@chanijjani Audio manager should check this.
```
audio_manager_result_t set_audio_stream_in(int channels, int sample_rate, int format) {
     config.format = get_audio_pcm_format(format);
      ...
}

int  get_audio_pcm_format(int format)
{
   switch(format) {
    case AUDIO_PCM_BIT_S16_LE :
         return PCM_FORMAT_S16_LE;
    case AUDIO_PCM_BIT_S32_LE :
         return PCM_FORMAT_S32_LE;
     default :
         return AUDIO_MANAGER_INVALID_PARAM;
}